ffs/interfaces.go

```diff
@@ -5,12 +5,15 @@ import (
 	"io"
 	"math/big"
 
+	"github.com/filecoin-project/go-address"
 	"github.com/ipfs/go-cid"
 	"github.com/textileio/powergate/deals"
 )
 
 // WalletManager provides access to a Lotus wallet for a Lotus node.
 type WalletManager interface {
+	// MasterAddr returns the master address.
+	MasterAddr() address.Address
```

Comment:
This is needed to the ffs manages can get the master address if the config says to use it

Reply:
I guess we could also make the master address a parameter for creating the `Manager`. Let me know which you prefer.

Reply:
I think this is OK. The owner of the master addr is the Wallet manager.
This API should cover the case there isn't a master addr, since that situation is possible. 
Might be good to explain in the comments that's a possible case, and explain might return a default value? (yet to see in the review)